index.d.ts

```diff
@@ -37,6 +37,7 @@ export interface YouTubeProps {
   id?: string;
   className?: string;
   containerClassName?: string;
+  iframeAttrs?: Record<string, string | number>;
```

Comment:
This does not implement any attribute removals or updates for when the component updates, as I don't know how or whether you'd prefer that in. Thoughts?